vue/src/components/imageViewer/imageUtils.ts

```diff
@@ -84,7 +84,7 @@ const drawData = (
             let imageWidth = rescale && poly.scaled ? poly.scaled.crop.width: imageDim[0];
             let imageHeight = rescale && poly.scaled ? poly.scaled.crop.height: imageDim[1];
             //Now if we have more image data than the scaled width and height are available we can take additional pixel information
-            if (rescale && poly.scaled) { //Implies S3/L8 data where more pixel data can be taken
+            if (rescale && poly.scaled) { //Implies S3/L8/PL data where more pixel data can be taken
```

Comment:
Another one of my Typos.
```suggestion
            if (rescale && poly.scaled) { //Implies S2/L8/PL data where more pixel data can be taken
```